pkg/install/resources.go

```diff
@@ -138,13 +138,18 @@ func ClusterRoleBinding(namespace string) *rbacv1.ClusterRoleBinding {
 }
 
 func Namespace(namespace string) *corev1.Namespace {
-	return &corev1.Namespace{
+	ns := &corev1.Namespace{
 		ObjectMeta: objectMeta("", namespace),
 		TypeMeta: metav1.TypeMeta{
 			Kind:       "Namespace",
 			APIVersion: corev1.SchemeGroupVersion.String(),
 		},
 	}
+
+	ns.Labels["pod-security.kubernetes.io/enforce"] = "privileged"
+	ns.Labels["pod-security.kubernetes.io/enforce-version"] = "latest"
+
```

Comment:
Do we also want to add `"pod-security.kubernetes.io/warn"` and `"pod-security.kubernetes.io/audit"` labels ?

Reply:
Do you mean adding warn and audit labels to the baseline and the restricted police like this example?
https://kubernetes.io/docs/tasks/configure-pod-container/enforce-standards-namespace-labels/#requiring-the-baseline-pod-security-standard-with-namespace-labels

Reply:
@shubham-pampattiwar 
Could help to take look?

Reply:
Yeah makes, we can skip adding the warm and audit labels

Reply:
@shubham-pampattiwar 
You are right. warn and audit labels are also needed.
I didn't meet the problem in the GKE and Kubeadm-created cluster, because the PSA default setting didn't trigger it, but VMware's TKG has a more restricted default setting, the expected warning is shown. 

https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/2.3/tkg-deploy-mc/mgmt-deploy-config-ref.html#pod-security-standards-for-pod-security-admission-controller-4 

https://github.com/kubernetes/enhancements/tree/master/keps/sig-auth/2579-psp-replacement#defaulting

Reply:
@blackpiglet Cool, created a PR to add these https://github.com/vmware-tanzu/velero/pull/6769